src/utils/request-api.js

```diff
@@ -6,7 +6,6 @@ const isNode = require('detect-node')
 const ndjson = require('ndjson')
```

Comment:
While you are at it, wanna rename this file to `send-request`?

Reply:
Done. In 6f25c3f56888f534882747370efef729155859e5